src/Elastic.CommonSchema/Serialization/BaseJsonFormatter.Generated.cs

```diff
@@ -19,9 +19,9 @@ internal partial class BaseJsonConverter<TBase> : EcsJsonConverterBase<TBase>
 		where TBase : Base, new()
 	{
 		private static bool ReadProperties(
-			ref Utf8JsonReader reader,
-			TBase ecsEvent,
-			ref DateTimeOffset? timestamp,
+			ref Utf8JsonReader reader, 
```

Comment:
Extra space, code cleanup should catch this but wonder how these were introduced (not important right now)